src/components/click-progression-game.js

```diff
@@ -1,5 +1,8 @@
 import { Screen } from "../../node_modules/genie/src/core/screen.js";
 import { gmi } from "../../node_modules/genie/src/core/gmi/gmi.js";
+import { settingsChannel } from "../../node_modules/genie/src/core/settings.js";
+import * as signal from "../../node_modules/genie/src/core/signal-bus.js";
```

Comment:
or this?

Reply:
Snipped out, another case of automatic import but no automatic import cleanup